### PR TITLE
docs: add PB blurb to v1.22.0 blogpost

### PR DESCRIPTION
**Pull request by huanjani**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v122.en.md [hunk 1]
```diff
@@ -33,6 +33,19 @@ Copilot v1.22 brings several new features and improvements:
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## IAM Role Permissions Boundary
+Whether you have an AWS Organizations Service Control Policy that requires an attached permissions boundary for IAM role creation, or simply want to want to add some guardrails to your application, Copilot can help. By using the `--permissions-boundary` flag with the `copilot app init` command, you can specify your existing IAM policy. That policy will get attached to any and all IAM roles that Copilot creates (within that application) as a permissions boundary. 
+  
+If you init your application like this:
+```console
+copilot app init --permissions-boundary examplePermissionsBoundaryPolicy
+```
+Copilot will attach the permissions boundary to every IAM role created in the app like this:
```
Comment (iamhopaul123) [inline site/content/blogs/release-v122.en.md]: Maybe
If you init your application with a permission boundary specified:
```console
copilot app init --permissions-boundary examplePermissionsBoundaryPolicy
```
The permissions boundary will be attached to every IAM role created in the app:

File: site/content/blogs/release-v122.en.md [hunk 2]
```diff
@@ -33,6 +33,19 @@ Copilot v1.22 brings several new features and improvements:
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## IAM Role Permissions Boundary
+Whether you have an AWS Organizations Service Control Policy that requires an attached permissions boundary for IAM role creation, or simply want to want to add some guardrails to your application, Copilot can help. By using the `--permissions-boundary` flag with the `copilot app init` command, you can specify your existing IAM policy. That policy will get attached to any and all IAM roles that Copilot creates (within that application) as a permissions boundary.
```
Comment (efekarakus) [inline site/content/blogs/release-v122.en.md]:
```suggestion
Whether you have an AWS Organizations Service Control Policy that requires an attached permissions boundary for IAM role creation, or simply want to want to add some guardrails to your application, Copilot can help. By using the `--permissions-boundary` flag with the `copilot app init` command, you can specify an existing IAM policy name. That policy will get attached to any and all IAM roles that Copilot creates (within that application) as a permissions boundary. 
```

File: site/content/blogs/release-v122.en.md [hunk 3]
```diff
@@ -33,6 +33,19 @@ Copilot v1.22 brings several new features and improvements:
...
 ## IAM Role Permissions Boundary
+Whether you have an AWS Organizations Service Control Policy that requires an attached permissions boundary for IAM role creation, or simply want to want to add some guardrails to your application, Copilot can help. By using the `--permissions-boundary` flag with the `copilot app init` command, you can specify your existing IAM policy. That policy will get attached to any and all IAM roles that Copilot creates (within that application) as a permissions boundary. 
+  
+If you init your application like this:
+```console
+copilot app init --permissions-boundary examplePermissionsBoundaryPolicy
+```
+Copilot will attach the permissions boundary to every IAM role created in the app like this:
+```yaml
+ExampleIAMRole
```
Comment (efekarakus) [inline site/content/blogs/release-v122.en.md]:
```suggestion
ExampleIAMRole:
```

**Review by efekarakus — APPROVED** [review]
Feel free to remove the label once addressed

File: site/content/blogs/release-v122.en.md (hunk 2 above)
Comment (dannyrandall) [inline site/content/blogs/release-v122.en.md]:
```suggestion
Whether you have an AWS Organizations Service Control Policy that requires an attached permissions boundary for IAM role creation, or simply want to add some guardrails to your application, Copilot can help. By using the `--permissions-boundary` flag with the `copilot app init` command, you can specify your existing IAM policy. That policy will get attached to any and all IAM roles that Copilot creates (within that application) as a permissions boundary. 
```

**Review by dannyrandall — APPROVED** [review]

File: site/content/blogs/release-v122.en.md (hunk 3 above)
Comment (huanjani) [reply site/content/blogs/release-v122.en.md]: Ooh good catch! Thanks!

**Pull request merged**